hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineModule.java

```diff
@@ -243,7 +244,9 @@ private HubEidasAttributeQueryRequestBuilder getHubEidasAttributeQueryRequestBui
             entityToEncryptForLocator,
             signatureAlgorithm,
             digestAlgorithm,
-            hubEntityId);
+            hubEntityId,
+            hubEidasEntityId.orElseThrow(
+                        () -> new InvalidConfigurationException(EIDAS_HUB_ENTITY_ID_NOT_CONFIGURED_ERROR_MESSAGE)));
```

Comment:
There is another case of using the `orElseThrow` block in this Module, so services with eidas integration should have `hubEidasEntityId` configured already.